test/specifications/gamification.feature

```diff
@@ -0,0 +1,72 @@
+# feature/gamification. feature

```

Comment:
This feature don't belong here!
move it in the gamification addon 
https://corlewsolutions.com/articles/article-127-how-to-setup-cucumber-in-a-rails-engine

Reply:
done